Dockerfile

```diff
@@ -77,6 +76,7 @@ ENV PYTHONDONTWRITEBYTECODE=1
 ENV RETICULATE_PYTHON="/root/.virtualenvs/r-tensorflow/bin/python"
 
 # Finally, apply any locally defined patches.
+ADD patches/ /tmp/patches/
```

Comment:
Instead of adding all the files, consider only adding the `kaggle_bigquery.R` file to a specific location better than /tmp/patches.